core/src/avm1/object/stage_object.rs

```diff
@@ -563,6 +563,9 @@ impl<'gc> DisplayPropertyMap<'gc> {
         property_map.add_property("_xmouse", x_mouse, None);
         property_map.add_property("_ymouse", y_mouse, None);
 
+        // Not part of the published spec?!
+        property_map.add_property("_lockroot", lock_root, Some(set_lock_root));
+
```

Comment:
This property map is designed for properties that can use GetProperty/SetProperty and apply to every DisplayObject; it doesn't seem like `_lockroot` is like this and is instead a normal property only for MovieClip (despite the `_` name). So this should be moved to `MovieClip::create_proto`.

Reply:
Thanks @Herschel! Makes sense. I have it working again, just need to clean it up a bit and will update soon.